### PR TITLE
[features] open-source the crate navitia-poi-model

### PR DESCRIPTION
**Pull request by patochectp**
Ref. ND-1480

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# `navitia-poi-model`
+
+`navitia-poi-model` defines the Rust model for [Navitia POI] with tools to
```
Comment (woshilapin) [inline README.md:3]: Usually, this goes with a corresponding link at the bottom.

```rust
[Navitia POI]: https://confluence.kisio.org/x/85Ui
```

The problem here is that we're open-sourcing something that was not, and the documentation here is not publicly accessible. Do we have any public documentation about the POI model for Navitia?

File: src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+// details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>
+
+//! Data structures and functions to manipulate Points of Interest (POIs)
+
+#![doc(html_root_url = "https://docs.rs/navitia-poi-model/0.1.0")]
```
Comment (woshilapin) [inline src/lib.rs:17]: I'd say this is probably useless.

**Review by woshilapin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (patochectp) [reply README.md]: No, I didn’t find any.
I removed link in efb6c77557496bd353d6612e4cd7c3661f8eae51

File: src/lib.rs (hunk 2 above)
Comment (patochectp) [reply src/lib.rs]: Done in efb6c77557496bd353d6612e4cd7c3661f8eae51

**Review by woshilapin — APPROVED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[package]
+authors = ["Kisio Digital <team.coretools@kisio.com>"]
+categories = ["data-structures", "parser-implementations"]
+description = "Navitia's POIs (Point of Interest) model"
+edition = "2018"
+keywords = ["data", "osm", "transit"]
+license = "AGPL-3.0-only"
+name = "navitia-poi-model"
+readme = "README.md"
+repository = "https://github.com/CanalTP/tartare-tools.git"
```
Comment (ArnaudOggy) [inline Cargo.toml:10]:
```suggestion
repository = "https://github.com/CanalTP/navitia-poi-model.git"
```

Comment (patochectp) [reply Cargo.toml]: Done

**Review by ArnaudOggy — APPROVED** [review]

**Pull request merged**